lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -19,6 +22,8 @@
     'info',
 )
 
+log = logging.getLogger("edx.sneakpeek_middleware")
```

Comment:
Understood. Did not add the lint line, but I did break up the longer one for the sake of improved readability.